bin/package.xml.in

```diff
@@ -55,5 +55,11 @@ necessary to build a fully-functional MongoDB driver.
   </required>
  </dependencies>
  <providesextension>mongodb</providesextension>
- <extsrcrelease/>
+ <extsrcrelease>
+  <configureoption name="with-libbson" default="no" prompt="Use system libbson"/>
+  <configureoption name="with-libmongoc" default="no" prompt="Use system libmongoc"/>
+  <configureoption name="with-openssl-dir" default="yes" prompt="OpenSSL dir for MongoDB"/>
```

Comment:
Should we add a note that "yes" results in automatic detection?

Reply:
Using "yes" (the binary) would also mean: system libbson, system libmongoc, and system-chiphers. I don't think we should recommend that?